### PR TITLE
Enforce camel case.

### PR DESCRIPTION
**Pull request by Levino**

### CODE REVIEW

**Comment by Levino** [comment]
@kiliw Please take over

**Comment by kiliw** [comment]
> @kiliw Please take over

Done.

**Comment by kiliw** [comment]
@Levino @Bakkhos @EmCeeEs please have a quick review. Its just a renaming of the constants to camel case.

**Comment by Levino** [comment]
Storybook does not builld.

File: package.json [hunk 1]
```diff
@@ -97,4 +97,4 @@
     "reactstrap": "^8.1.1",
     "typescript": "^3.6.4"
   }
-}
+}
```
Comment (Levino) [inline package.json]:
```suggestion
}

```

**Pull request merged**